api/envoy/config/core/v3/protocol.proto

```diff
@@ -171,6 +172,13 @@ message AlternateProtocolsCacheOptions {
 
   // Allows pre-populating the cache with entries, as described above.
   repeated AlternateProtocolsCacheEntry prepopulated_entries = 4;
+
+  // Optional list of hostnames suffixes for which Alt-Svc entries can be shared. For example, if
+  // this list contained the value `.c.example.com`, then an Alt-Svc entry for `foo.c.example.com`
```

Comment:
@phlax this is a place we could do with better guidance on canonical formatting as style guide.

Reply:
yep - not sure - i would say these are literals - defo not italics tho - below uses single quotes - i would say they are all literal